autospec/specfiles.py

```diff
@@ -1162,3 +1163,35 @@ def _write(self, string):
 
     def _write_strip(self, string):
         self.specfile.write_strip(string)
+
+    @staticmethod
+    def quote_filename(filename):
+        """
+        Quotes the filename, if necessary. Identifies and skips any RPM directive prefix.
+        """
+        # Characters that require quoting -- only those with special
+        # meaning in specfiles
+        special_chars = set(" \t")
+        # Build up the output as a string
+        quoted = ''
+        # Separate any directive prefix
+        # This will break if your directive contains a slash!
+        line = filename.split("/", 1)
+        if len(line) < 1:
+            print_fatal("Filenames must start with a / ({0})".format(filename))
+            sys.exit(1)
+        elif len(line) > 1:
+            # Keep the prefix
+            quoted += line[0]
+            # Filename is remaining part
+            filename = line[1]
+        else:
```

Comment:
I'm concerned about this logic.
```
len("/some/filename".split("/", 1))  # -> 2, hits the elif and quoted contains an empty string
len("%doc /some/filename".split("/", 1))  # -> 2, hits the elif and quoted contains %doc
len("some/filename".split("/", 1))  # -> 2, hits the elif, quoted contains empty string **but** there is no leading /
len("filename".split("/", 1))  # -> 1, hits the else, filname gets the correct value.
len("".split("/", 1))  # -> 1, hits the else, filename is empty, but looks like filename turns into "/" below...
```
When would `len(line)` ever be < 1? It doesn't look like this is actually testing that the file starts with a "/"